### PR TITLE
Make evoke cross platform

### PR DESCRIPTION
**Pull request by JulienGrv**
Hi,

I'd also like to remove the Makefile to only support the CMakelists.txt since it works for both Windows and Linux. Would it be okay if I do it? I will subsequently update the README.md, .travis.yml, and so on.

### CODE REVIEW

File: fw/include/fw/filesystem.hpp [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+
+namespace filesystem
+{
+using namespace boost::filesystem;
+using boost::system::error_code;
+}
+
 #else
+
 #    include <filesystem>
```
Comment (dascandy) [inline fw/include/fw/filesystem.hpp]: Does not work on Travis, sadly. Can you re-disable this code? To be picked up separately.

**Comment by dascandy** [comment]
Thanks for the other changes - looks like you cleaned up a lot of accidental boost filesystem use.

File: fw/include/fw/filesystem.hpp (hunk 1 above)
Comment (JulienGrv) [reply fw/include/fw/filesystem.hpp]: Yes I saw this, it is because link to libstdc++fs is missing for filesystem. I will look into this

**Comment by JulienGrv** [comment]
All green now

**Comment by dascandy** [comment]
Thanks!

**Pull request merged**